docs/guides/troubleshooting.md

```diff
@@ -1,3 +1,8 @@
+----
+-layout: default
+-title: Troubleshooting
+----
```

Comment:
Didn't you mention that the front matter only accepts `---`, and not `----`?

```suggestion
---
layout: default
title: Troubleshooting
---
```

Reply:
failure to push